src/utils/time.ts

```diff
@@ -0,0 +1,54 @@
+import { yellow } from "../../deps.ts";
+
+export function renderTimeDelta(delta: number): string {
```

Comment:
Looks like a good candidate to add a unit test for

Reply:
done in https://github.com/denoland/deployctl/pull/283/commits/2644ac82e4a58b71824edd3b35a03bdc227bff8c